test/smoke/automation/src/editors.ts

```diff
@@ -15,29 +15,27 @@ export class Editors {
         }
     }
 
-    public async selectTab(tabName: string, untitled: boolean = false, isEditor?: boolean): Promise<void> {
-        await this.code.waitAndClick(`.tabs-container div.tab[aria-label="${tabName}, tab"]`);
-        if (isEditor) {
-            await this.waitForEditorFocus(tabName, untitled);
-        }
+    public async selectTab(fileName: string): Promise<void> {
```

Comment:
isEditor variable is needed here since it helps to divide Expo webview tabs from editor ones.